route53_crossfeed_app.tf

```
@@ -281,7 +281,7 @@ resource "aws_route53_record" "crossfeed_staging_MX" {
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
-resource "aws_route53_record" "crossfeed_staging_TXT" {
+resource "aws_route53_record" "crossfeed_staging_TXT1" {
```

I think this should be modified to align with the resource naming scheme we have used in this file:
```suggestion
resource "aws_route53_record" "crossfeed_staging_ses_TXT" {
```

---

My bad @lfilip-ivaal - I didn't notice that we already have a `crossfeed_staging_ses_TXT` in this file.  I think that this resource name can stay the same as it was before:
```
resource "aws_route53_record" "crossfeed_staging_TXT" {
```

---

Resolved with https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/80/commits/dfe350d7cf8a4f5423162c02cc356f6bc9560b93.